templates/etc/systemd/system/docker-run.erb

```diff
@@ -39,6 +39,9 @@ ExecStart=/usr/bin/<%= @docker_command %> run \
         <% if @command %> <%= @command %><% end %>
 <%- if @before_stop %>ExecStop=-<%= @before_stop %>
 <% end -%>
+<% if @stop_before_kill -%>
+ExecStop=-/usr/bin/<%= @docker_command %> stop <% if @stop_timeout != false %>-t <%= @stop_timeout %> <% end %><%= @sanitised_title %>
+<%- end -%>
 ExecStop=-/usr/bin/<%= @docker_command %> kill <%= @sanitised_title %>
```

Comment:
Yes, that line with docker kill should be removed. But we have to wait what garethr say because that change seems to be done intentionally. On the other hand when you look into initscript template, you'll notice that there is no docker kill...